composer.json

```diff
@@ -15,20 +15,21 @@
     ],
 
     "require": {
-        "php": "^5.5|^7.0",
-        "symfony/options-resolver": "^2.7|^3.0|^4.0|^5.0"
+        "php": "^5.5|^7.0|^8.0",
+        "symfony/options-resolver": "^2.7|^3.0|^4.0|^5.0|^6.0"
     },
 
     "require-dev": {
         "doctrine/cache":               "^1.5",
         "doctrine/common":              "^2.5",
         "doctrine/orm":                 "^2.5",
-        "phpunit/phpunit":              "^4.8.35|^5.4.0|^6.0|^7.0",
+        "phpunit/phpunit":              "^8.1",
         "silex/silex":                  "^1.3",
-        "symfony/dependency-injection": "^2.7|^3.4|^4.0|^5.0",
-        "symfony/config":               "^2.7|^3.4|^4.0|^5.0",
-        "symfony/yaml":                 "^2.7|^3.4|^4.0|^5.0",
-        "twig/twig":                    "^1.23|^2.0"
+        "symfony/dependency-injection": "^2.7|^3.4|^4.0|^5.0|^6.0",
+        "symfony/config":               "^2.7|^3.4|^4.0|^5.0|^6.0",
+        "symfony/yaml":                 "^2.7|^3.4|^4.0|^5.0|^6.0",
+        "twig/twig":                    "^1.23|^2.0",
```

Comment:
I think at least PHP 5.5 and Symfony 2 and 3, maybe 4, could be dropped.  I don't know if testing in PHP 5 is even possible.  Actually, the void return type isn't supported until PHP 7.1.